### DISCUSSION TITLE
Support for Open AI ADA embeddings?

### DISCUSSION THREAD

**Discussion by karrtikiyer-tw** [marqo-ai/marqo#784] [Q&A] [answered]
how do I store openAI or cohere embeddings in Marqo Vector Store? Any tutorials or examples?

**Comment by pandu-k** [comment, marked as answer]
We don't support connecting to these APIs directly from within Marqo. However you can use the [custom vector object](https://docs.marqo.ai/2.11/Guides/Advanced-Usage/document_fields/?h=own+vectors#custom-vector-object) to add vectors that you generate from these APIs into Marqo
